docs/setup-gradle.md

```diff
@@ -769,6 +783,21 @@ The `init-script` supports several additional configuration parameters that you
 | GRADLE_PLUGIN_REPOSITORY_USERNAME    |  | the username for the repository URL to use when resolving the Develocity and CCUD plugins                                                                               |
 | GRADLE_PLUGIN_REPOSITORY_PASSWORD    |  | the password for the repository URL to use when resolving the Develocity and CCUD plugins; Consider using secrets to pass the value to this variable                    |
 
+All additional configuration parameters can be specified as input actions using:
```

Comment:
Please change the main definition table to show the input parameters, and have a separate table showing the alternative env vars. (Effectively switching the order of columns here).

Reply:
Main definition table showing the input parameters. 
The additional table switched order of the columns
https://github.com/gradle/actions/pull/244/commits/eec8c4dc24f3cad4ad887ab4c76607ec88039941